bot/skills/kozula.py

```diff
@@ -0,0 +1,47 @@
+import logging
+import xml.etree.ElementTree as ET
+import requests
+from mode import cleanup_queue_update
+
+from telegram import Update
+from telegram.ext import Updater, CommandHandler, CallbackContext
+
+logger = logging.getLogger(__name__)
+
+KOZULA_RATE = 600000
```

Comment:
Should we place variable in env?

Reply:
Heh, kozula updated so infrequently. Not such a big deal.